src/analysis/Analyses.ts

```diff
@@ -520,12 +514,12 @@ export const BaseAnalyses = (
         .Workspaces.workspace(workspaceInfo.namespace, workspaceInfo.name)
         .listActiveFileTransfers();
       setActiveFileTransfers(!_.isEmpty(fileTransfers));
-
-      await refreshRuntimes();
     });
-
-    load();
-  });
+    if (workspace?.workspaceInitialized) {
+      load();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [workspace, workspaceInfo.name, workspaceInfo.namespace]);
```

Comment:
The dependencies we're ignoring are things like `refreshApps` that cause a render loop if included.